ecpp

```diff
@@ -0,0 +1,130 @@
+#!/usr/bin/python
+
+from __future__ import print_function
+import ConfigParser
+import sys
+import StringIO
+from verify import *
+
+#oneseventwoeight = 1728
+
+try:
+    xrange
+except NameError:
+    xrange = range
+
+# dict with params
+def par_set():
+    global params
+    params = {'s':0,'w':0,'j':0,'t':'0','a':0,'b':0,'q':0,'r':0}
+
+# division function
+def division(n,i):
+    if n % i == 0:
+        raise ValueError("not prime: " + str(n))
+
+with open(sys.argv[1], 'r+') as f:
+    config_buff = f.read()
+    f.close()
+
+config_ini = StringIO.StringIO(config_buff)
+config = ConfigParser.RawConfigParser(allow_no_value=True)
+config.readfp(config_ini)
+sections = config.sections()
+
+par_set()
+
+# combinations
+c_t1 = ['s','w','j','t']
+c_t2 = ['s','w','a','b','t']
+llr_t = ['s','q']
+bls_t = ['s','b']
+
+# prime candidate
+n = config.get('Candidate','N')
+print(n)
```

Comment:
the code is printing `n` twice, once here and then in line 50, the difference is that here it doesn't say why or what it is printing...